pkgs/by-name/lu/lunarvim/package.nix

```
@@ -0,0 +1,93 @@
+{ fd
+, fetchFromGitHub
+, fzf
+, git
+, lib
+, makeWrapper
+, neovim
+, nodePackages
+, python311
+, ripgrep
+, stdenv
+, tree-sitter
+}:
+let
+  lunar-python = python311.withPackages ( p: with p; [ pynvim ] );
+  runtimeDeps = [
+    fd
+    fzf
+    git
+    neovim
+    ripgrep
+    lunar-python
+    tree-sitter
+  ] ++ (with nodePackages; [
+    neovim
+  ]);
+in
+stdenv.mkDerivation rec {
+  pname = "lunarvim";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "LunarVim";
+    repo = "LunarVim";
+    rev = version;
+    sha256 = "sha256-z1Cw3wGpFDmlrAIy7rrjlMtzcW7a6HWSjI+asEDcGPA=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir $out
+
+
+    # Install required files
+    export LUNARVIM_BASE_DIR="$out/base"
+    mkdir $LUNARVIM_BASE_DIR
+    cp init.lua $LUNARVIM_BASE_DIR
+    cp -r {lua,snapshots} $LUNARVIM_BASE_DIR
+
+    # Create laucher script
+
+    ## Copy template to where the installer script expects it
+    mkdir -p $LUNARVIM_BASE_DIR/utils/bin/
+    cp ./utils/bin/lvim.template $LUNARVIM_BASE_DIR/utils/bin/lvim.template
+
+    ## Required by the installer - will replace values in the template
+    export INSTALL_PREFIX="$out"
+    export XDG_DATA_HOME='"$HOME/.local/share"'
+    export XDG_CACHE_HOME='"$HOME/.cache"'
+    export XDG_CONFIG_HOME='"$HOME/.config"'
+
+    bash ./utils/installer/install_bin.sh
+    mv $out/bin/lvim $out/bin/.lvim-wrapped
+    makeWrapper "$out/bin/.lvim-wrapped" "$out/bin/lvim" \
+      --prefix PATH : ${lib.makeBinPath runtimeDeps }
+
+    ## Remove the template after installation
+    rm -rf $LUNARVIM_BASE_DIR/utils
+
+    # Copy desktop icons
+    mkdir -p $out/share/icons/
+    cp -r utils/desktop $out/share/icons/hicolor
+    rm $out/share/icons/hicolor/lvim.desktop
+
+    # Create desktop launcher
+    mkdir -p $out/share/applications/
+    cp utils/desktop/*.desktop $out/share/applications/
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "An IDE layer for Neovim with sane defaults";
+    homepage = "https://www.lunarvim.org/";
+    license = licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ loveisgrief ];
```

You should add the `mainProgram` attribute so that e.g. `lib.getExe` works properly.
```suggestion
    maintainers = with lib.maintainers; [ loveisgrief ];
    mainProgram = "lvim";
```